release-calendar/src/app.ts

```diff
@@ -13,12 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+import 'reflect-metadata';
 import {createConnection, Connection} from 'typeorm';
 import {Release} from './entities/release';
+import {addingTestData} from './development/developmentData';
```

Comment:
Can we move `developmentData.ts` into the test directory? Because all of the files in `src/` are going to get deployed for production, but we ignore `test/`.

Reply:
Oh! I didn't know that. Sounds good!